src/Controller/ForgotPasswordController.php

```diff
@@ -107,7 +115,7 @@ public function checkEmail(SessionInterface $session): Response
         $session->remove(self::SESSION_CAN_CHECK_EMAIL);
 
         return $this->render('forgot_password/check_email.html.twig', [
-            'tokenLifetime' => self::LIFETIME_HOURS,
+            'tokenLifetime' => ResetPasswordTimeHelper::getFormattedSeconds($this->resetPasswordHelper->getTokenLifetime()),
```

Comment:
This can be added to the trait in PR #6 similar to how session methods are done..